### ISSUE TITLE
[Bug]: Check if Image Auto Format is enabled before asking Engine itself

### ISSUE THREAD

**Issue by solverat** [pimcore/pimcore#15340]
### Pimcore version

10.x, 11.x

### Steps to reproduce

This Condition needs to be switched.

https://github.com/pimcore/pimcore/blob/dd654dde67eed57f4c27436b1dca2bd3042da1b6/models/Asset/Image/Thumbnail/Config.php#L824

to
```php
if ($autoFormatConfig['enabled'] && Model\Asset\Image\Thumbnail::supportsFormat($autoFormat)) {}
```

On some instances, the `checkFormatSupport` will crash the whole server, if the format is not supported.

PR can be provided easily, but I'm not sure which branch, is `10.6` still open for bugs?

**Comment by dkarlovi** [comment]
@solverat 10.5 is still open for bugs.

But, 
> On some instances, the checkFormatSupport will crash the whole server, if the format is not supported.

seems like a bigger bug.

**Comment by solverat** [comment]
@dkarlovi :smile: That's right. It's an imagick version/configuration issue, however - if the configuration says "no", no test image should be written.

**Comment by brusch** [comment]
@solverat `10.5` is still open for PRs - looking forward to your PR 😊 Thanks

**Comment by dvesh3** [comment]
Fixed by #15408 @solverat thanks for the fix.
